### ISSUE TITLE
Doesn't work well with Reactotron — restoring redux snapshots throws error

### ISSUE THREAD

**Issue by cihadturhan** [infinitered/ignite-ir-boilerplate#173]
First of all, thanks for providing such a great boilerplate. I couldn't decide if I open this issue here, reactotron or reactotron-redux.

I can successfully create redux snapshots on reactotron. However, I have the following error on the screen:
```
MyError: The push method cannot be invoked on an Immutable data structure.

This error is located at:
    in LaunchScreen (created by Connect(LaunchScreen))
    in Connect(LaunchScreen) (at SceneView.js:31)
    in SceneView (at CardStack.js:394)
...
```
I dig into the reactotron code and realized it's because of `react-navigation` doesn't use immuatable.js
Before restoring snapshot: 
```
store: {
   nav: {...}, //plain JS object
   github: {...}, // Immutable
   search: {...} // Immutable
  ...
}
```
After restoring the snapshot:
```
store: {
   nav: {...}, // Immutable
   github: {...}, // Immutable
   search: {...} // Immutable
  ...
}
```
This happens because reactotron-redux uses `onRestore()` function to restore state and makes every object `Immutable` and `react-navigation` tries to push a new item to an array inside `nav`. This crashes the app app and I can't be able to use the app.

What I wonder is, is there a way to make `react-navigation` Immutable or is there a way to say `seamless-immutable` that nav object should not be immutable?

**Comment by cihadturhan** [comment]
Yeah, I saw reactotron is highly configurable and didn't miss the opportunity to hack 

**If you want to make reactotron snapshots work with this boilerplate**, make the following change in `ReactotronConfig.js`
```diff
-.use(reduxPlugin({ onRestore: Immutable }))
+.use(reduxPlugin({ onRestore: (state, oldState) =>
+  Object.entries(state)
+    .reduce((prev, [key, value]) => key === 'nav' ? { ...prev, [key]: value } : {
+       ...prev,
+         [key]: Immutable(value),
+       }, {}),
+}))
```

It's somewhat hack so I believe there could be a better solution, so I won't open a PR but let me know if it's accepteble anyway.

**Comment by jamonholmgren** [comment]
@cihadturhan I think that hack is fine in the ReactotronConfig.js file. Would you want to open a PR for it?
